vm/src/sliceable.rs

```diff
@@ -208,6 +208,22 @@ impl<T: Clone> PySliceableSequenceMut for Vec<T> {
     }
 }
 
+pub trait PySliceableSequenceOwner {
+    fn owner_type() -> &'static str;
```

Comment:
You can make this an associated constant - `const OWNER_TYPE: &'static str;`. Then the macro can be `<$type as $crate::pyobject::PyClassDef>::NAME`